packages/components/src/combobox-control/index.js

```diff
@@ -135,7 +135,7 @@ function ComboboxControl( {
 	const onFocus = () => {
 		setIsExpanded( true );
 		onFilterValueChange( '' );
-		setInputValue( '' );
+		// setInputValue( '' );
```

Comment:
```suggestion

```

Reply:
Might as well just remove it instead of commenting out :) We've got git!

Reply:
Whoops, that was unintended! I don't think removing it changes anything, but it's not needed for the fix so I'll revert the change.